### ISSUE TITLE
Uncaught TypeError: invalid pid

### ISSUE THREAD

**Issue by Reen90** [ioquatix/script-runner#76]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.18.0 x64
**Electron**: 1.3.15
**OS**: Mac OS X 10.12.5
**Thrown From**: [script-runner](https://github.com/ioquatix/script-runner) package 2.1.1

### Stack Trace

Uncaught TypeError: invalid pid

```
At internal/process.js:155

TypeError: invalid pid
    at process.kill (internal/process.js:155:13)
    at ScriptRunnerProcess.module.exports.ScriptRunnerProcess.stop (/packages/script-runner/lib/script-runner-process.coffee:28:15)
    at ScriptRunner.killProcess (/packages/script-runner/lib/script-runner.coffee:56:24)
    at /packages/script-runner/lib/script-runner.coffee:88:10
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom 2.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom 2.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at Pane.module.exports.Pane.destroyItem (/Applications/Atom 2.app/Contents/Resources/app/src/pane.js:673:28)
    at TabBarView.module.exports.TabBarView.closeTab (/Applications/Atom 2.app/Contents/Resources/app/node_modules/tabs/lib/tab-bar-view.js:340:32)
    at Object.TabBarView.addElementCommands.tabs:close-tab (/Applications/Atom 2.app/Contents/Resources/app/node_modules/tabs/lib/tab-bar-view.js:92:32)
    at HTMLUListElement.commandsWithPropagationStopped.(anonymous function) (/Applications/Atom 2.app/Contents/Resources/app/node_modules/tabs/lib/tab-bar-view.js:83:42)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom 2.app/Contents/Resources/app/src/command-registry.js:265:35)
    at CommandRegistry.handleCommandEvent (/Applications/Atom 2.app/Contents/Resources/app/src/command-registry.js:3:65)
    at CommandRegistry.module.exports.CommandRegistry.dispatch (/Applications/Atom 2.app/Contents/Resources/app/src/command-registry.js:166:25)
    at AtomEnvironment.module.exports.AtomEnvironment.dispatchContextMenuCommand (/Applications/Atom 2.app/Contents/Resources/app/src/atom-environment.js:1320:34)
    at EventEmitter.outerCallback (/Applications/Atom 2.app/Contents/Resources/app/src/application-delegate.js:347:31)
    at emitThree (events.js:116:13)
    at EventEmitter.emit (events.js:194:7)
```

### Commands

```
     -6:30.7.0 script:run (input.hidden-input)
  5x -6:25.9.0 core:move-down (input.hidden-input)
  6x -6:22.2.0 core:move-up (input.hidden-input)
 21x -6:19.3.0 core:move-down (input.hidden-input)
     -6:13 script:close-view (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-one-dark-syntax.theme-one-dark-ui)
     -5:14.5.0 core:confirm (input.hidden-input)
     -1:10.8.0 run:script (input.hidden-input)
     -0:10.7.0 tabs:close-tab (div.title)
```

### Non-Core Packages

```
script 3.15.0 
script-runner 2.1.1 
```

**Comment by ioquatix** [comment]
We need to know how to reproduce.

**Comment by ioquatix** [comment]
This is probably fixed in the latest v2.2.0 release, please open a new issue if you still have problems.
